File: src/main/scala/com/example/examplemod/entities/ModTileEntities.java
package com.example.examplemod.entities;

import com.example.examplemod.ExampleMod;

import net.minecraft.util.ResourceLocation;
import net.minecraftforge.fml.common.registry.GameRegistry;

public class ModTileEntities {
	public static void register() {
		GameRegistry.registerTileEntity(TileEntityExpDrainMachine.class, new ResourceLocation(ExampleMod.MODID + ":tiles/exp_drain_machine"));
	}
}

File: src/main/scala/org/burbokop/exp_craft/entities/ModTileEntities.scala
package org.burbokop.exp_craft.entities;

import org.burbokop.exp_craft.ExpCraftMod;

import net.minecraft.util.ResourceLocation;
import net.minecraftforge.fml.common.registry.GameRegistry;

object ModTileEntities {
	def register() =
		GameRegistry.registerTileEntity(classOf[TileEntityExpDrainMachine], new ResourceLocation(ExpCraftMod.MOD_ID + ":tiles/exp_drain_machine"));
}
